src/components/structural/header/Header.js

```diff
@@ -133,21 +133,24 @@ class Header extends Component {
         // Bind to keyboard to listen for shortcuts
         document.addEventListener("keydown", this.handleKeyDown.bind(this));
 
-        // Warn the issue before refreshing the page
-        try {
-            let editor = window.ace.edit("ace-editor");
-            editor.getSession().on("change", () => {
-                let text = editor.getSession().getValue();
-                if (this.props.text !== text) {
-                    this.setState({ editorChange: true });
-                } else {
-                    this.setState({ editorChange: false });
-                }
+        if (window.location.href !== window.origin + "/reference") {
```

Comment:
Let's do this with layout types instead. Check the reference page to see how it's done there.

Reply:
Actually, it might not be the reference page. It might be in the header. Search for `layout_types.REFERENCE` (I think that's what I named it).